android/src/main/java/id/animo/mdocdatatransfer/MdocDataTransferModule.kt

```diff
@@ -18,7 +18,7 @@ class MdocDataTransferModule : Module() {
             MdocDataTransferEvent.ON_RESPONSE_SENT
         )
 
-        Function("initialize") {
+        Function("initialize") { serviceName: String ->
```

Comment:
Servicename seems unused?

Reply:
Kept it to have matching interfaces, can remove it though.